jigasi/rootfs/defaults/sip-communicator.properties

```diff
@@ -11,16 +11,28 @@ net.java.sip.communicator.impl.protocol.sip.acc1=acc1
 {{ if and .Env.JIGASI_SIP_PORT .Env.JIGASI_SIP_TRANSPORT }}
 net.java.sip.communicator.impl.protocol.sip.acc1.PROXY_ADDRESS={{ .Env.JIGASI_SIP_SERVER }}
 net.java.sip.communicator.impl.protocol.sip.acc1.PROXY_AUTO_CONFIG=false
-net.java.sip.communicator.impl.protocol.sip.acc1.PROXY_PORT={{ .Env.JIGASI_SIP_PORT }}
-net.java.sip.communicator.impl.protocol.sip.acc1.PREFERRED_TRANSPORT={{ .Env.JIGASI_SIP_TRANSPORT }}
+net.java.sip.communicator.impl.protocol.sip.acc1.PROXY_PORT={{ .Env.JIGASI_SIP_PORT | default "5060" }}
+net.java.sip.communicator.impl.protocol.sip.acc1.PREFERRED_TRANSPORT={{ .Env.JIGASI_SIP_TRANSPORT | default "UDP" }}
+{{ end }}
+{{ if and .Env.JIGASI_ENABLE_SDES_SRTP | default "0" | toBool }}
+net.java.sip.communicator.impl.protocol.sip.acc1.SAVP_OPTION=1
+net.java.sip.communicator.impl.protocol.sip.acc1.DEFAULT_ENCRYPTION=true
+net.java.sip.communicator.impl.protocol.sip.acc1.DEFAULT_SIPZRTP_ATTRIBUTE=false
+net.java.sip.communicator.impl.protocol.sip.acc1.ENCRYPTION_PROTOCOL.ZRTP=0
+net.java.sip.communicator.impl.protocol.sip.acc1.ENCRYPTION_PROTOCOL.SDES=1
+net.java.sip.communicator.impl.protocol.sip.acc1.ENCRYPTION_PROTOCOL.DTLS-SRTP=2
```

Comment:
since we are here we should also make the use of DTLS configurable perhaps?

Reply:
I have no environment for DTLS, for now only SDES

Reply:
then lets set it to 0 then and disable AVPF

Reply:
do you mean  DTLS-SRTP=0 and SAVP_OPTION=false?

Reply:
changed DTLS options
- ENCRYPTION_PROTOCOL.DTLS-SRTP=0
- ENCRYPTION_PROTOCOL_STATUS.DTLS-SRTP=fasle

Reply:
exactly